src/test/java/com/google/devtools/build/lib/sandbox/cgroups/CpuTest.java

```diff
@@ -52,16 +52,18 @@ public void getCpuLimit_v1() throws IOException {
 
   @Test
   public void setCpuLimit_v2() throws IOException {
-    File limit = scratch.file("cgroup/cpu/cpu.max", "-1 100000").getPathFile();
+    File limit = scratch.file("cgroup/cpu/cpu.max", "-1 100000\n").getPathFile();
```

Comment:
Instead of changing the content here, would you please add a new test with this test case? (I assume both cases can happen)

Maybe we can name the new test as `fileWithLineBreak_ setCpuLimit_v2()`.

Reply:
Done! TBH, I don't think both formats are possible, probably the newline is always present, but testing both does not harm